lms/djangoapps/discussion_api/views.py

```diff
@@ -144,6 +144,9 @@ class ThreadViewSet(_ViewMixin, DeveloperErrorViewMixin, ViewSet):
         * following: If true, retrieve only threads the requesting user is
             following
 
+        * view: "unread" for threads the requesting user has not read, or
+            "unanswered" for question threads with no marked answer.
+
         The topic_id, text_search, and following parameters are mutually
```

Comment:
Greg is correct. I mistook mutually exclusive as 'can only select one of "unanswered" or "unread"'. View can be used with the others.